Documentation/GeneralConventions/ReviewPolicy.rst

```diff
@@ -0,0 +1,201 @@
+.. include:: /Includes.rst.txt
```

Comment:
4 space indent

Reply:
Would you like to edit&correct directly, as outlined in the review policy? 😅😬